docs/source/serving/openai_compatible_server.md

```diff
@@ -219,6 +222,20 @@ it works better with vLLM.
 
 Recommended flags: `--tool-call-parser llama3_json --chat-template examples/tool_chat_template_llama3_json.jinja`
 
+#### IBM Granite
+
+Supported models:
+* `ibm-granite/granite-20b-functioncalling`
```

Comment:
Probably better to swap the order of these and move the granite 3.0 one to the top